configs/train_configs/sft/tulu3_8b_preview_mix.yaml

```diff
@@ -0,0 +1,29 @@
+# Note, the exact model was trained on TPUs in a different repo
+# Example file for mixing, use exact tulu 2 mixture for replication allenai/tulu-v2-sft-mixture
+model_name_or_path: meta-llama/Meta-Llama-3-8B
+model_revision: main
+use_flash_attn: true
+tokenizer_name: meta-llama/Meta-Llama-3-8B
+use_slow_tokenizer: true
+dataset_mixer:
+  allenai/tulu-v2-sft-mixture: 1.0
+  HuggingFaceH4/no_robots: 1.0
+  meta-math/MetaMathQA: 0.25
+  m-a-p/CodeFeedback-Filtered-Instruction: 1.0
+  ai2-adapt-dev/daring-anteater-specialized: 1.0
```

Comment:
did we want to commit this config? Don't have a strong opibion either way.

Reply:
Yes and no -- this will be our current canonical preview mix config, but I'll update this before submitting the PR to follow what we talked about today in the tulu standup